.teamcity/_self/projects/WPComTests.kt

```diff
@@ -72,128 +70,6 @@ object WPComTests : Project({
 	buildType(P2E2ETests)
 })
 
-fun gutenbergCoreE2eBuildType(): BuildType {
-	return BuildType ({
-		id("WPComTests_gutenberg_core_e2e")
-		name = "Gutenberg Core E2E Tests"
-		description = "Runs Gutenberg core E2E tests against a Dotcom environment."
-
-		artifactRules = """
-			gutenberg/artifacts => artifacts
-			logs/*.log => logs
-		""".trimIndent()
-
-		vcs {
-			root(Settings.WpCalypso)
-			cleanCheckout = true
-		}
-
-		params {
-			// WP.com URL of the site to test against.
-			password("WP_BASE_URL", "credentialsJSON:5cc9ce44-c31a-4591-9f02-cda749351bff");
-			// WP.com username and password to use for logging in.
-			password("WP_USERNAME", "credentialsJSON:ab140672-6955-4206-9ae4-df940896992d");
-			password("WP_PASSWORD", "credentialsJSON:1b787674-1c6f-41c5-9b39-41768fa1aa0c");
-			// Calypso client ID and secret for remote logging in.
-			password("WP_CLIENT_ID", "credentialsJSON:7bcd18c5-7ebe-42ab-9f85-45abcea3f21b");
-			password("WP_CLIENT_SECRET", "credentialsJSON:87a99f9c-2bf6-43c2-bd43-903f28bec4fb");
-			// Application password for authenticating REST API requests.
-			password("WP_APP_PASSWORD", "credentialsJSON:2f191dbd-7341-4ff9-acab-f5dd0111e364");
```

Comment:
If those secrets aren't in use, perhaps it will make sense to clean them up from the encrypted secrets. Worth double-checking if that's the case though.

Reply:
According to the FG (PCYsg-vnR-p2) these encrypted passwords should live inside `settings.kts`, but I didn't find them. Do you know where do we set them @tyxla?

Reply:
This section of the docs applies only if you're creating a new password and want to reference it in the TeamCity config. 

The actual passwords live encrypted in a file in the `calypso-e2e` package - for accessing them, see the decrypting section in the post you linked.

Reply:
Actually, my bad: these are set in the parameters section of the build you're deleting - done in the TC interface.